docs/wiki/99 - Developers/Alternative Install/README.md

```diff
@@ -0,0 +1 @@
+Instructions for alternative installations of our software
```

Comment:
I don't understand this file or why it's here.

Reply:
Rida added a placeholder README in each of the new wiki directories to describe what they're for.

It can be deleted in this PR

Reply:
@driusan @johnsaigle I removed it.